docs/v2tov3.md

```diff
@@ -147,6 +147,63 @@ For a complete view of the GoSDK `v3`, please refer to [TODO: link API and tutor
 * Some member names changed, e.g. `MetricSet` is now `metric.Set`
 * Added `EntityMetadata` struct to fulfill the [JSON `v2` schema](#json-v2-schema-changes).
 
+## Migrate from `v2` to `v3` steps
+### Imports
+* First of all we've renamed some packages. So the first you can do when migrating from `v2` to `v3` is run these commands inside your 
+integration src folder :
+
+```bash
+find . -type f -iname "*.go" |xargs -I {} sed -i '' 's/github.com/newrelic/infra-integrations-sdk/sdk/github.com/newrelic/infra-integrations-sdk/integration/' {}
```

Comment:
Just a minor about the sed statement. With sed, you can use any separator char, `/` is the standard but in some cases like this one I feel it is very confusing. 
You could replace it by: 
`sed -i '' 's#github.com/newrelic/infra-integrations-sdk/sdk#github.com/newrelic/infra-integrations-sdk/integration#'`

Reply:
👍